pkg/Library/NodeGraph/Editor.js

```diff
@@ -13,15 +13,11 @@ update({pipeline, selectedNodeKey}, state) {
     state.selectedPipelineName = pipeline?.$meta.name;
     // new pipeline, choose a selectedNode if there isn't one
     if (!selectedNodeKey) {
-      return {selectedNodeKey: pipeline?.nodes?.[0]?.key};
+      return {selectedNodeKey: keys(pipeline?.nodes)?.[0]?.key};
```

Comment:
I think this is not quite right? The zeroth key is `keys(pipeline?.nodes)[0]` or `keys(pipeline?.nodes.pop())`.

Reply:
fixed, thank you!